--- before
+++ after
@@ -28,7 +28,7 @@
 $GOROOT/src/os/exec_unix.go
 $GOROOT/src/os/executable.go
 $GOROOT/src/os/executable_darwin.go
-$GOROOT/src/os/file.go
+$WORK/b049/file.go
 $GOROOT/src/os/file_open_unix.go
 $GOROOT/src/os/file_posix.go
 $GOROOT/src/os/file_unix.go
@@ -60,3 +60,4 @@
 $GOROOT/src/os/wait_unimp.go
 $GOROOT/src/os/zero_copy_posix.go
 $GOROOT/src/os/zero_copy_stub.go
+$WORK/b049/toyhook_linkname.go
